app/webpack.common.ts

```diff
@@ -38,6 +34,9 @@ const commonConfig = {
             options: {
               useBabel: true,
               useCache: true,
+              compilerOptions: {
+                noEmit: false, // override from tsconfig.json
```

Comment:
@j-f1 I've dropped this override in 9810d3d as I couldn't find any problems with either `yarn && yarn build:dev && yarn start` or `yarn && yarn build:prod && yarn start:prod` the resulting bundles. Let me know if there's some context here that I've missed.